### ISSUE TITLE
[Bug]: Dragon Corpses Shown As Dragon Models

### ISSUE THREAD

**Issue by KrissyMC7** [Alex-the-666/Ice_and_Fire#5297]
### What happened?

After a lot of experimenting between a friend and I, we discovered a bug in multiplayer servers where the corpses of the dragons would display as the default dragon models standing upright. This does not occur in singleplayer worlds and cannot determine the cause of this bug.
![image](https://github.com/AlexModGuy/Ice_and_Fire/assets/172064605/91508760-d071-4c3a-b20d-01debae3049e)

### Minecraft Version

1.20.1

### Ice and Fire version

iceandfire-2.1.13-1.20.1-beta-4

### Citadel version

citadel-2.5.4-1.20.1

### Relevant log output

```shell
N/A
```

### Other Information

- [X] I am using other mods besides Ice and Fire
- [X] I have attached a log file from the session that I have experienced this issue

**Comment by Dannieboi** [comment]
Yeah i have the same issue in my modpack with dragon corpses as well as all dragons in the world are just super tiny

**Comment by Dannieboi** [comment]
also only happens server side

**Comment by Dannieboi** [comment]
you ever figured out what other mod caused this to happen?

**Comment by KrissyMC7** [comment]
> you ever figured out what other mod caused this to happen?

Unfortunately no, we had many mods to work with during that time but the bug didn't drastically affect our gameplay.
